lang/test/org/partiql/lang/ast/passes/StaticTypeRewriterTests.kt

```diff
@@ -859,8 +862,8 @@ class StaticTypeRewriterTests : RewriterTestBase() {
         // We always pass the query under test through all of the basic rewriters primarily because we need
         // FromSourceAliasVisitorTransform to execute first but also to help ensure the queries we're testing
         // make sense when they're all run.
-        val defaultRewriters = basicRewriters(ion)
-        val originalExprNode = defaultRewriters.rewriteExprNode(parse(tc.originalSql))
+        val defaultTransforms = basicVisitorTransforms()
+        val originalExprNode = defaultTransforms.transformStatement(parse(tc.originalSql).toAstStatement()).toExprNode(ion)
 
         val rewrittenExprNode = try {
             rewriter.rewriteExprNode(originalExprNode)
```

Comment:
Uses of `Rewriter`/`rewritten` will be changed in the `StaticTypeRewriter` migration [PR](https://github.com/partiql/partiql-lang-kotlin/pull/318).